examples/mcp-server-sandbox/client.py

```
@@ -0,0 +1,236 @@
+            print(f"[host] list_blobs (after restart) -> {after}")
+
+            read_back = _result_payload(await session.call_tool(
+                "read_blob", {"name": BLOB_NAME},
+            ))
+            print(f"[host] read_blob({BLOB_NAME!r}) -> {read_back}")
+            return read_back
+
+
+def _patch_operating_mode(mode: str) -> None:
+    patch = json.dumps({"spec": {"operatingMode": mode}})
+    subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)